lib/fastlane/plugin/versioning/actions/get_app_store_version_number.rb

```diff
@@ -16,7 +16,11 @@ def self.run(params)
           bundle_id = GetInfoPlistValueAction.run(path: plist, key: 'CFBundleIdentifier') # TODO: add same kind of flag to support build setting variables
         end
 
-        uri = URI("http://itunes.apple.com/lookup?bundleId=#{bundle_id}")
+        if params[:country]
+          uri = URI("http://itunes.apple.com/lookup?bundleId=#{bundle_id}&country=#{country}")
+        else
+          uri = URI("http://itunes.apple.com/lookup?bundleId=#{bundle_id}")
+        end
```

Comment:
any chance of writing a little test for this? there is some sort of example of these inside of `get_app_store_version_number_spec.rb` - and if you aren't sure how to run the tests - cd into your local `fastlane-plugin-versioning` repo checkout, and run `rake spec`

Reply:
I've added some new commits to the pull request. One of them now contains a test case, which runs `get_app_store_version_number` with the new `country` parameter.

This includes a response I've downloaded from the itunes lookup API and added to the project. The new test case and response mockup handling should be in line with existing test cases.

There is just one problem remaining when running the new test case: The JSON parser throws an error, when parsing the mock response from the itunes lookup API. _I've verified that the JSON is **valid** with multiple validators._
Unfortunately I'm out of ideas on how to proceed. I even updated the JSON gem to the latest version. Could you take a look?